getting_started/step_by_step/7_room_two.rst

```diff
@@ -191,11 +200,56 @@ when you click the door to leave the pub.
 
 Configure the change between rooms
 ----------------------------------
+
 What we now need is to tell Escoria to move the character to the correct
 location when they change between rooms. For this we will use a script that
 will run when the room first loads. It will check which room the player has
 come from and immediately send them to the correct ``ESCLocation``.
 
+Setup and Ready events
+^^^^^^^^^^^^^^^^^^^^^^
+
+To finish the switch between rooms, we need to explain how the room change
+works. When you activate an ``ESCExit`` (or ``ESCItem`` with "is exit"
+selected), you use the ``change_scene`` command to tell Escoria to replace the
+current room with new room. It doesn't tell the Escoria anything about how to
+set up the new room or where to put the player - this is the responsibility
+of the new room.
+
+Each room's root node is an ``ESCRoom`` node, and the script attached to this
+(if there is one) is responsible for setting up anything dynamic in the room.
+When ``change_scene`` loads the room, it will look in the new room's script for
+a ``setup`` event. Any commands you put in this script will run before the
+player sees the new room - use this to reset any movable objects to their start
+positions.
+
+After the ``setup`` event completes, a transition will run (there is a default
```

Comment:
...if "automatic transitions" in `change_scene` is true, anyway.

Reply:
Do we want to add that? I think it's probably going to confuse things for new users. If you feel adding it in is important then my suggestion would be another hint section below with the other hints. Something like :
```
.. hint::

    Transitions run by default but this change be manually overridden as part of the ``change_scene`` command.
```

Reply:
I agree and like what you've recommended (although maybe change it slightly: `Transitions are run by default...`).